examples/dog-mode/AppManifest.json

```diff
@@ -31,7 +31,7 @@
         {
             "type": "grpc-interface",
             "config": {
-                "src": "https://github.com/eclipse/kuksa.val.services/blob/v0.2.0/hvac_service/proto/sdv/edge/comfort/hvac/v1/hvac.proto",
+                "src": "https://raw.githubusercontent.com/eclipse/kuksa.val.services/v0.2.0/hvac_service/proto/sdv/edge/comfort/hvac/v1/hvac.proto",
                 "required": {
                     "methods": [
                         "SetAcStatus",
```

Comment:
should we remove that completely, since we have the service already defined in the custom model?

Reply:
Seems to make sense - we could actually also remove the VSS reference, it isn't used